### PR TITLE
[JavaScript] standardize inheritance: UCR

### PR DESCRIPTION
**Pull request by lwyszomi**
Hi @orangejenny 

I sent the next part of javascript changes, currently, I refactored UCR.

Please let me know if you have any questions.

Regards,
Łukasz

### CODE REVIEW

File: corehq/apps/userreports/static/userreports/js/expression_evaluator.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /* globals hqDefine */
 hqDefine('userreports/js/expression_evaluator', function () {
-    var ExpressionModel = function (editor, submitUrl, initialData) {
+    var expressionModel = function (editor, submitUrl, initialData) {
         var self = this;
```
Comment (orangejenny) [inline corehq/apps/userreports/static/userreports/js/expression_evaluator.js]: `var self = {};` and then `return self;` at the end

**Review by orangejenny — CHANGES_REQUESTED** [review]
Just one comment, the rest looks great.

**Comment by orangejenny** [comment]
Hi Łukasz, one more issue: when I edit a report builder report locally (http://localhost:8000/a/DOMAIN/reports/builder/edit/ID/), this error comes up. Can you check it out?

screen shot 2019-01-15 at 6 05 48 pm

**Comment by lwyszomi** [comment]
@orangejenny updated

**Review by orangejenny — APPROVED** [review]

**Pull request merged**